packages/parser-sass/README.md

```diff
@@ -0,0 +1,62 @@
+# SASS (`@hint/parser-sass`)
+
+The `SASS` parser is built on top of [PostCSS][postcss] so hints can
+analyze `SASS` and `SCSS` files. It emits the same events as
+`@hint/parser-css` so existing hints targeting `CSS` files will work
+without modification.
+
+To use it you will have to install it via `npm`:
+
+```bash
+npm install @hint/parser-sass
+```
+
+Note: You can make `npm` install it as a `devDependency` using the
+`--save-dev` parameter, or to install it globally, you can use the
```

Comment:
> or to install it globally

Did we decide to go against global installs? I believe that individual packages installed globally will not work, only if they are in a configuration...
Do we have a task to update all the docs to remove this?

Reply:
That seems to be the growing consensus. No existing task to go forth and change the docs yet, though. I'm on board if you want to create one.